thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -605,10 +585,14 @@ class CIndexReadSlaveBase : public CSlaveActivity
             translators.kill();
             keyManagers.kill();
             keyMergerManager.clear();
+            contextLogger.reset();
+            // NB: think wouldn't need this if contextLogger was *not* reset
+            // because if re-executing, stop() was called and it is already in sync with contextLogger stats.
```

Comment:
is this true? If so, we should remove both contextLogger.reset() and statsUpdater.resetStart()